loops-and-arrays.js

```diff
@@ -57,5 +67,12 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 11 is 1331'
 // ]
 
+console.log('Opdracht 3:');
+
+for (let i = 0; i < squares.length; i++) {
+    const volume = squares[i] * squares[i] * squares[i];
+    console.log(`Het volume van ${squares[i]} is ${volume}`);
+}
+
```

Comment:
Hetzelfde eigenlijk als eerder, je logt de aangepaste waarde wel in de console, maar de originele array wordt niet aangepast.

for (let i = 0; i < squares.length; i++) {
  const volume = squares[i] * squares[i] * squares[i];

  squares[i] = "Het volume van " + squares[i] + " is " + volume;
}